frontend/packages/process-editor/src/components/ConfigPanel/EditDataType/SelectDataType/SelectDataType.tsx

```diff
@@ -22,33 +23,39 @@ export const SelectDataType = ({
 }: SelectDataTypeProps): React.ReactElement => {
   const { t } = useTranslation();
   const { mutateDataType } = useBpmnApiContext();
-  const handleChangeDataModel = (dataModelId?: string) => {
-    if (dataModelId === existingDataType) return;
-    const dataTypeChange: DataTypeChange = {
-      newDataType: dataModelId,
-      connectedTaskId: connectedTaskId,
-    };
-    mutateDataType(dataTypeChange);
+  const currentValue = existingDataType ? [existingDataType] : [];
+  const handleChangeDataModel = (newDataModelIds?: string[]) => {
```

Comment:
I think we should use the word `datamodel` instead of `dataModel` as it is one word 😄 
```suggestion
  const handleChangeDatamodel = (newDatamodelIds?: string[]) => {
```

Reply:
Goes for entire file/package too 😅 Tried to change some of them in my previous PR 😄